History.md

```diff
@@ -17,26 +17,51 @@ pattern.
 
 ### Migrating from version 0.x
 
-`cheerio.load( html[, options ] )`` This method continues to produce
-jQuery-like functions, bound to the provided input text. In prior releases, the
-provided string was interpreted as a document fragment. This meant that in a
-statement such as:
+`cheerio.load( html[, options ] )` This method continues to act as a "factory"
+function. It produces functions that define an API that is similar to the
+global `jQuery` function provided by the jQuery library. The generated function
+operates on a DOM structure based on the provided HTML.
+
+In releases prior to version 1.0, the provided HTML was interpreted as a
+document fragment. Following version 1.0, strings provided to the `load` method
+are interpreted as documents. The same example will produce a `$` function that
+operates on a full HTML document, including an `<html>` document element with
+nested `<head>` and `<body>` tags. This mimics web browser behavior much more
+closely, but may require alterations to existing code.
+
+For example, the following code will produce different results between 0.x and
+1.0 releases:
 
     var $ = cheerio.load('<p>Hello, <b>world</b>!</p>');
 
-The resulting `$` function would operate on a tree whose root element was a
-paragraph tag.
+    $.root().html();
 
-With this release of Cheerio, strings provided to the `load` method are
-interpreted as documents. The same example will produce a `$` function that
-operates on a full HTML document, including an `<html>` document element with
-nested `<head>` and `<body>` tags. This mimics web browser behavior much more
-closely, but may require alterations to existing code. To work with fragments,
-first load an empty document, and then use the resulting `$` to parse the
-input:
+    //=> In version 0.x: '<p>Hello, <b>world</b>!</p>'
+    //=> In version 1.0: '<html><head></head><body><p>Hello, <b>world</b>!</p></body></html>'
 
+Users wishing to parse, manipulate, and render full documents should not need
```

Comment:
This is much clearer - in other words, the breaking change isn't `.load()`, it's the *combination* of `.load()` and `.root()`.